stores/dotenv/store.go

```diff
@@ -0,0 +1,152 @@
+package dotenv //import "go.mozilla.org/sops/stores/dotenv"
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/base64"
+	"encoding/gob"
+	"fmt"
+	"go.mozilla.org/sops"
+	"go.mozilla.org/sops/stores"
+	"strings"
+)
+
+const SopsPrefix = "sops_"
+
+// Store handles storage of dotenv data
+type Store struct {
+}
+
+func (store *Store) LoadEncryptedFile(in []byte) (sops.Tree, error) {
+	branch, err := store.LoadPlainFile(in)
+	if err != nil { return sops.Tree{}, err }
+
+	resultBranch := make(sops.TreeBranch, 0)
+	metadata := stores.Metadata{}
+	md_found := false
```

Comment:
Go uses camelCase for local variables, so `mdFound` or even `metadataFound` would be better here. I personally prefer longer names, but many Go programmers prefer shorter. Up to you.